ios/Handlers/RNFlingHandler.h

```diff
@@ -0,0 +1,12 @@
+//
```

Comment:
Can you please delete all this comment. We don't have any file headers in the project for now (may want to add some in the future).

Reply:
Ok, I was just trying to be coherent with another files in the project